lib/board.rb

```diff
@@ -75,4 +75,13 @@ def place(ship, coordinates)
         @cells[coordinate].place_ship(ship)
     end
   end
+
+  def render(show_ship = false)
+    board_render = "  1 2 3 4 \n" +
+    "A #{cells["A1"].render(show_ship)} #{cells["A2"].render(show_ship)} #{cells["A3"].render(show_ship)} #{cells["A4"].render(show_ship)} \n" +
+    "B #{cells["B1"].render(show_ship)} #{cells["B2"].render(show_ship)} #{cells["B3"].render(show_ship)} #{cells["B4"].render(show_ship)} \n" +
+    "C #{cells["C1"].render(show_ship)} #{cells["C2"].render(show_ship)} #{cells["C3"].render(show_ship)} #{cells["C4"].render(show_ship)} \n" +
+    "D #{cells["D1"].render(show_ship)} #{cells["D2"].render(show_ship)} #{cells["D3"].render(show_ship)} #{cells["D4"].render(show_ship)} \n"
+    board_render
```

Comment:
I hard-coded this method instead of iterating over the cells for both readability and because we're both shooting for iteration 3 completion at this point, and aren't focused on having the board render at any size.

Reply:
This is great. It's organized and I like how the cells are laid out. It makes sense to me and it's "easy" to read.